lib/jekyll/commands/clean.rb

```diff
@@ -17,13 +18,16 @@ def init_with_program(prog)
 
         def process(options)
           options = configuration_from_options(options)
-          destination = options['destination']
-          metadata_file = File.join(options['source'], '.jekyll-metadata')
-          sass_cache = File.join(options['source'], '.sass-cache')
+          destination = options["destination"]
+          metadata_file = File.join(options["source"], ".jekyll-metadata")
+          sass_cache = File.join(options["source"], ".sass-cache")
 
+          # Disabling rubocop due to https://github.com/bbatsov/rubocop/issues/1451
+          # rubocop:disable Style/HashSyntax
           remove(destination, checker_func: :directory?)
           remove(metadata_file, checker_func: :file?)
           remove(sass_cache, checker_func: :directory?)
```

Comment:
That should still be a hash rocket.  Since technically it consumes a hash and it is a hash until it's assigned it should be treated as a named hash regardless.  A distinction used to be made, but nowdays since it's technically a hash when it comes in, make it follow hash rules. See:

```
[7] pry(main)> def hello(world: nil)
[7] pry(main)*   $stdout.puts world
[7] pry(main)* end  
=> :hello

[8] pry(main)> hello({ :world => "hello" })
hello
=> nil

[9] pry(main)> hello(:world => "hello")
hello
=> nil

[10] pry(main)> hello(world: "hello")
hello
=> nil
```